sdk/dotnet/Pulumi/Deployment/GrpcEngine.cs

```diff
@@ -3,6 +3,7 @@
 using System.Collections.Generic;
 using System.Threading.Tasks;
 using Grpc.Core;
```

Comment:
This isn't the old package reference still being used via the Tools dep is it?

Reply:
I think so, i will remove it